content/en/docs/concepts/storage/volumes.md

```diff
@@ -934,7 +934,7 @@ A container using a Secret as a [`subPath`](#using-subpath) volume mount will no
 receive Secret updates.
 {{< /note >}}
 
-For more details, see [Configuring Secrets](/docs/concepts/configuration/secret/).
+As a default, Secrets are readable only by `root`. For more details, see [Configuring Secrets](/docs/concepts/configuration/secret/).
```

Comment:
What is this?

Reply:
@tengqm Part of this: https://github.com/kubernetes/website/issues/38436

Reply:
We should definitely check that the assertion is accurate.

Reply:
@sftim is there anything we have add more on this?